packages/binding.template/src/templating.ts

```diff
@@ -21,19 +25,21 @@ import {
     templateEngine
 } from './templateEngine'
 
+import type { TemplateEngine, TemplateOptions } from './templateEngine'
+
 import {
   anonymousTemplate as AnonymousTemplate
 } from './templateSources'
 
-var _templateEngine
+var _templateEngine : TemplateEngine
 const cleanContainerDomDataKey = domData.nextKey()
 
-export function setTemplateEngine (tEngine) {
+export function setTemplateEngine (tEngine : TemplateEngine | undefined) : void {
```

Comment:
Better: `tEngine? : TemplateEngine`

Reply:
Correction: This isn't an optional parameter; instead, the TemplateEngine can be set to "undefined." But this case is only relevant for testing... The type can be left as is for now.